VogtPayroll3/Employee.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Linq;
+
+namespace VogtPayroll3
+{
+    class Employee
+    {
+        // Fields
+        private string _firstName;
+        private string _lastName;
+        private int _empID;
+        private int _hoursWorked;
+        private decimal _payrate;
+        private const decimal taxAmount = .30m;
+
+        public string FirstName { get => _firstName; set => _firstName = value; }
+        public string LastName { get => _lastName; set => _lastName = value; }
```

Comment:
This is the same thing again -- theres no reason to have 2 fields -- one private/one public. Either have a public property or have a private property, but make the decision based on logically whether the propreties need to be exposed.